src/ztc/datamodel/tests/test_eigenschap_model.py

```diff
@@ -0,0 +1,69 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from django.utils import timezone
+
+
+from ..models import Eigenschap
+
+from .factories import (
+    EigenschapFactory, EigenschapReferentieFactory, EigenschapSpecificatieFactory,
+    ZaakTypeFactory
+)
+
+
+class EigenschapModelTests(TestCase):
+    def test_model_raises_error_when_both_specificatie_and_referentie_are_set(self):
+        zaaktype = ZaakTypeFactory.create()
+        specificatie = EigenschapSpecificatieFactory.create()
+        referentie = EigenschapReferentieFactory.create()
+
+        eigenschap = EigenschapFactory.create(
+            definitie="definitie",
+            eigenschapnaam="aard product",
+            toelichting="nieuw / verandering / ambtshalve wijziging / ontheffing / intrekking",
+            specificatie_van_eigenschap=specificatie,
+            referentie_naar_eigenschap=referentie,
+            is_van=zaaktype
```

Comment:
When using factories, you don't need to specify this explicitly if it's defined as a subfactory